docker/lite/build.sh

```diff
@@ -78,9 +84,9 @@ chmod -R o=g lite
 # Build vitess/lite image
 
 if [[ -n "$flavor" ]]; then
-	docker build --no-cache -f Dockerfile.$flavor -t vitess/lite:$flavor .
+  docker build --no-cache -f Dockerfile.$flavor -t vitess/lite:$flavor .
```

Comment:
Done.

I've put $lite_image here and also defined a variable for `Dockerfile(.$flavor)`. This way I could get rid of the `if` here.